src/platforms/php/common/integrations.mdx

```diff
@@ -90,14 +82,13 @@ This integration decides whether an event should not be captured according to a
 
 ### ModulesIntegration
 
-This integration logs with the event details all the versions of the packages installed with Composer; the root project is included too.
-
+This integration logs all the versions of the packages installed with Composer with the event details; the root project is also included.
 
-## Customize Integrations
+## Custom Integrations
 
-It is also possible to customize the list of integration without disabling the default integration using the `integrations` option and passing a callable.
+You can customize the list of integration without disabling the default integration using the `integrations` option and passing a callable.
```

Comment:
While it's true that you can pass a `callable` if you have advanced needs, you can also pass a fixed list of integrations that you want to use which should be enough for most of the use cases